docker/stable/Dockerfile.cpu

```diff
@@ -21,8 +21,10 @@ RUN apt-get install -y \
     texlive-latex-base \
     texlive-latex-extra
 
-# upgrade essential packages
-RUN pip install --upgrade pip setuptools==51.3.3
```

Comment:
other 3 files are replacing notebook install, this is replacing setuptools. is it intended?

Reply:
oops wrongly delete this line. nice catch